regions/desktop/rating_stats_card.py

```diff
@@ -0,0 +1,68 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+
+class RatingStats(Region):
+
+    _addon_title_locator = (By.CSS_SELECTOR, '.AddonTitle > a')
+    _addon_image_locator = (By.CSS_SELECTOR, '.AddonSummaryCard-header-icon-image')
+    _addon_author_locator = (By.CSS_SELECTOR, '.AddonTitle-author > a')
+    _addon_rating_locator = (By.CSS_SELECTOR, '.AddonSummaryCard-addonAverage')
+    _number_of_reviews_locator = (By.CLASS_NAME, 'RatingsByStar-count')
+    _rating_stars_locator = (By.CSS_SELECTOR, '.Rating-star')
+    _rating_by_star_locator = (By.CSS_SELECTOR, '.RatingsByStar-graph > a')
+
+    @property
+    def addon_title(self):
+        return self.find_element(*self._addon_title_locator)
+
+    def click_addon_title(self):
+        self.find_element(*self._addon_title_locator).click()
+        from pages.desktop.frontend.details import Detail
+
+        return Detail(self.selenium, self.page)
+
+    @property
+    def addon_image(self):
+        return self.find_element(*self._addon_image_locator)
+
+    def click_addon_image(self):
+        self.find_element(*self._addon_image_locator).click()
+        from pages.desktop.frontend.details import Detail
+
+        return Detail(self.selenium, self.page)
+
+    @property
+    def addon_author_names(self):
+        return [i.text for i in self.find_elements(*self._addon_author_locator)]
+
+    def click_author_name(self, index=0):
+        self.find_elements(*self._addon_image_locator)[index].click()
```

Comment:
Why limit this to `index = 0`. Shouldn't we want to click through all the authors present in the card? 
Also, `*self._addon_image_locator)[index].click()` doesn't look like the correct element here.  You are using the 'image'.

Reply:
index=0 is not a limit, is a default value. In case the method is called without specifying an index, its value will be 0 and the first author will be clicked.
I will change the locator with the right one.